content/docs/tutorials/get-started/data-access.md

```diff
@@ -1,31 +1,39 @@
 # Data Access
 
-We've seen how to [version data](/doc/tutorials/get-started/data-versioning) for
-sharing among team members or environments of the same project. But what about
-reusing your data and models from an existing DVC project in other projects, or
-on a production deployment?
+Okay, now that we've learned how to _track_ data and models with DVC and how to
+version them with Git, next question is how can we _use_ these artifacts outside
+of the project? How do I download a model to deploy it? How do I download a
+specific version of a model? How do I reuse datasets across different projects?
 
-<abbr>DVC repositories</abbr> serve as an entry point for your data. The
-following CLI commands and an API are available to access any version of it,
-from any machine where DVC is installed.
+> Especially, these questions come up very fast when you see the way DVC saves
+> files to the remote storage, e.g.
+> `s3://dvc-public/remote/get-started/fb/89904ef053f04d64eafcc3d70db673` 😱
+> instead of `model.pkl` or `data.xml`.
```

Comment:
```suggestion
> These questions tend to come up when you browse the files that DVC saves
> to remote storage, e.g.
> `s3://dvc-public/remote/get-started/fb/89904ef053f04d64eafcc3d70db673` 😱
> instead of the original files, name such as `model.pkl` or `data.xml`.
```

Reply:
Maybe link "remote storage"